src/Extensions/Shopware/Install/Services/Install/Release.php

```diff
@@ -110,7 +110,7 @@ public function installShopware(InstallationRequest $request)
     {
         $this->releaseDownloader->downloadRelease($request->getRelease(), $request->getInstallDir());
 
-        if ($request->getRelease() === 'latest' || version_compare($request->getRelease(), '5.1.2', '>=')) {
+        if ($request->getRelease() === 'latest' || version_compare($request->getRelease(), '5.2.12', '>=')) {
```

Comment:
Please double check if this must be changed. Looks like this is a conditional for old or newer ways of insalling.